packages/api/src/controllers/asset.ts

```diff
@@ -343,7 +361,7 @@ app.get("/:id", authorizer({}), async (req, res) => {
     );
   }
 
-  res.json(asset);
+  res.json(db.asset.cleanWriteOnlyResponse(asset));
```

Comment:
This change should be reverted as I explained in a previous comment. The `mung` middleware enforces this on every response.